pkg/lib/machine-add.html

```diff
@@ -24,8 +24,15 @@ <h4 class="modal-title" translate="yes">Add new host</h4>
       </div>
 
       <label translate="yes" class="control-label">User name</label>
-      <input class="form-control" id="add-machine-user"
-             type="text""/>
+      <div class="pf-c-input-group">
+        <input class="form-control" id="add-machine-user"
+               type="text"/>
+        <a tabindex="0" role="button" data-toggle="popover"
+           data-trigger="focus" data-placement="bottom" translate="data-content"
+           data-content="Leave blank to connect to this machine as the currently logged in user. If you enter a different username, that user will always be used when connecting to this machine.">
+          <span class="fa fa-lg fa-info-circle"></span>
```

Comment:
This ends up on a separate row for me for some reason, rather than next to the text entry as it does in your screenshot. Here is how it looks for me:
![Screenshot from 2020-11-03 15-59-47](https://user-images.githubusercontent.com/209895/98001209-aa817e80-1ded-11eb-8148-449cfd12fb7c.png)

In your screenshot it lacks padding between the input field and the icon, though.

Reply:
Ignore the weird transparency of the popup in the screenshot above, that it some flicker that occurs for a milisecond when I press the Print screen button on my keyboard

Reply:
[Patternfly spacers](https://www.patternfly.org/v4/guidelines/spacers)

Reply:
Looking at the [Patternfly documentation](https://www.patternfly.org/v4/components/form/design-guidelines#popovers), it seems this is the recommended syntax:
```
      <button class="pf-c-form__group-label-help" aria-label="More info">
        <i class="pficon pf-icon-help" aria-hidden="true"></i>
      </button>
```

That makes the icon consistent with that use on the Networking page (Bond dialog) and the Logs page.
The aria-label would also ensure it works on a screen reader.

Reply:
More popover docs: https://www.patternfly.org/v4/components/popover/design-guidelines

Reply:
Thanks @andreasn for promoting the PF4 here ;) 
Here is the html example for that.
https://www.patternfly.org/v4/components/form/html#vertically-aligned-labels

Reply:
I can't get them to work easily. Clicking the info button reloads the page.  The icon shows up as a missing glyph.
I propose we use the old style until we rewrite all the dialogs in React.

Reply:
Ok!